resources/quality/dagoma/dagoma_neva_magis_pla_fast.inst.cfg

```diff
@@ -0,0 +1,17 @@
+[general]
+version = 3
+name = Fast
+definition = dagoma_neva
+
+[metadata]
+setting_version = 4
+type = quality
+quality_type = draft
+weight = -2
+material = generic_pla
+
+[values]
+layer_height = 0.2
+
+material_print_temperature = =default_material_print_temperature + 10
+material_bed_temperature_layer_0 = =default_material_print_temperature + 10
```

Comment:
This is probably a mistake, the bed temperature must not depend on the print temperature. It should be `default_material_bed_temperature`

Reply:
Fixed in f1efaa7